index.html

```diff
@@ -437,6 +437,12 @@ <h3>Programming Diversity</h3>
 		<script src="vendor/popper.min.js"></script>
 		<script src="vendor/bootstrap.min.js"></script>
 		<script>
+			        $(document).ready(function () {
+    		$('.first-button').on('click', function () {
+  
+			$('.animated-icon').toggleClass('open');
+    		});
+		});
```

Comment:
The indentation of this block seems off, could you fix that? :)

Reply:
Thank You for your response.
Ok, I will start working on these changes very soon.

Reply:
Hello, I want to know one thing is I need to again clone this project and makes changes locally and then again PR or there is any other way of doing this.
Please guide me.

Reply:
Thanks, and sorry for the late reply! For the future, it’s always best to @-mention some people for review :)

Reply:
> Hello, I want to know one thing is I need to again clone this project and makes changes locally and then again PR or there is any other way of doing this.
> Please guide me.

You can simply add a commit to your fork and push it, the pull request should be updated. :)

Reply:
OK, Thanks

Reply:
Please tell me one more thing can I use vs code to indent it using alt+shift+f. This directly indents the code properly. and then I push the code. Or I do manually?

Reply:
Please check it, I indent this block properly or not.